pallets/schemas/src/lib.rs

```diff
@@ -193,6 +200,13 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_runtime_upgrade() -> Weight {
+			migrations::v1::migrate_schemas_with_additional_settings::<T>()
```

Comment:
it does seem like we will need a cleanup ticket to remove all these runtime upgrade logic

Reply:
yep https://github.com/LibertyDSNP/frequency/issues/1106